### ISSUE TITLE
tab "Tests" tests are not shown for concept exercises [by design] 

### ISSUE THREAD

**Issue by Reunina** [exercism/exercism#5790]
to continue the discussion started in https://github.com/exercism/elixir-test-runner/issues/56. 

The tests tab is not shown ( by design ) on concept exercices. but maybe we can provide the user why ( and that is not a bug )

**Comment by iHiD** [comment]
@taiyab Should we add the tests tab back in and have a nice message here?

**Comment by SleeplessByte** [comment]
Yes. I think adding the tab but having the reason why it's empty on concept exercises there is a very good idea.

**Comment by taiyab** [comment]
@iHiD @SleeplessByte Yea, let's add it back in, and write that blurb.
